CMakeLists.txt

```diff
@@ -190,15 +190,11 @@ IF (CMAKE_SYSTEM_NAME STREQUAL Linux)
     INSTALL(FILES vegastrike.xpm DESTINATION share/pixmaps)
     INSTALL(FILES vslogo.xpm DESTINATION share/pixmaps)
 
-    INSTALL(FILES vs DESTINATION bin PERMISSIONS
```

Comment:
not sure on removing this; but yes `vsettings` should certainly get swapped for `vega-settings` now.